airbyte-webapp/src/components/ToolTip/TooltipLearnMoreLink.tsx

```diff
@@ -0,0 +1,15 @@
+import { FormattedMessage } from "react-intl";
+
+import styles from "./TooltipLearnMoreLink.module.scss";
+
+interface TooltipLearnMoreLinkProps {
+  url: string;
+}
+
+export const TooltipLearnMoreLink: React.VFC<TooltipLearnMoreLinkProps> = ({ url }) => (
```

Comment:
Just curious, why did you use `React.VFC<>` here, but `React.FC<>` in the `ToolTip` component?

Reply:
VFC (Void FunctionComponent) is a type of component that does not allow children to be passed through it. It's something I discovered recently while working on this.

Reply:
Ah I see, and we don't want this component to have any children because it is just a simple div with a link